Makefile

```diff
@@ -213,9 +213,9 @@ endif
 
 define _show_endpoints
 # The following endpoints are available
-echo "http://$(if $(IS_WSL2),$(get_my_ip),127.0.0.1):9081                                                                 - oSparc platform"
-echo "http://$(if $(IS_WSL2),$(get_my_ip),127.0.0.1):18080/?pgsql=postgres&username=scu&db=simcoredb&ns=public  - Postgres DB"
-echo "http://$(if $(IS_WSL2),$(get_my_ip),127.0.0.1):9000                                                                 - Portainer"
+echo "http://$(if $(IS_WSL2),$(get_my_ip),127.0.0.1.nip.io):9081                                                                 - oSparc platform"
```

Comment:
MINOR: what does ``nip.io`` brings here ? I understand it can be useful to map dns but ... what difference make here?

Actually, should we just set ``$(get_my_ip)`` in all cases?? 
```bash
echo "http://$(get_my_ip):9081      
echo "http://$(get_my_ip):18080 /?pgsql=postgres&username=scu&db=simcoredb&ns=public
...
 ```
 MINOR: please align the enumeration at the end

Reply:
I have already explained here https://github.com/ITISFoundation/osparc-simcore/pull/1887#discussion_r630932248

I will also alright the output visualised in the terminal.